### PR TITLE
feat(api-client): add Postman import

### PR DESCRIPTION
**Pull request by Mason-Little**
**Overview**

Added native support for importing Postman collections through our standard import modal. Users can now seamlessly import Postman collections as if they were OpenAPI specs, with automatic detection and conversion.

- Added Postman collection detection and conversion
- Created postman.ts for handling Postman-specific functionality
- Enhanced import modal to handle both OpenAPI and Postman specs transparently
- Integrated @scalar/postman-to-openapi for reliable conversions

### CODE REVIEW

**Comment by hanspagel** [comment]
* lock file got a quick update, no deletes, we chill
* tightened up some code style, looks clean now
* tossed in a changeset

hmu if it’s sus or anything’s off

File: packages/api-client/src/libs/postman.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { convert } from '@scalar/postman-to-openapi'
+
+/** Checks if the given content is a Postman collection */
+export function isPostmanCollection(content: string): boolean {
+  try {
+    const parsed = JSON.parse(content)
+    const isPostman =
+      parsed.info?._postman_id !== undefined &&
+      parsed.info?.schema?.includes('schema.getpostman.com')
```
[the remark this answers is not in the document]
Comment (hanspagel) [reply packages/api-client/src/libs/postman.ts]: ✅ done

**Review by amritk — APPROVED** [review]
looks gooooooood

**Comment by antlio** [comment]
looks good! should we update the wording in the import modal accordingly?

**Review by marclave — CHANGES_REQUESTED** [review]
i converted petstore openapi -> postman collection then import but i dont have any servers

image

image

**Comment by marclave** [comment]
i converted Scalar Galaxy -> Postman collection then imported it and it fails

[Scalar Galaxy.postman_collection.json](https://github.com/user-attachments/files/18021573/Scalar.Galaxy.postman_collection.json)

**Comment by marclave** [comment]
@Mason-Little i updated the copy to make it searchable : )

image

image

image

**Comment by marclave** [comment]
> looks good! should we update the wording in the import modal accordingly?

done :)

**Comment by Mason-Little** [comment]
> i converted Scalar Galaxy -> Postman collection then imported it and it fails
> 
> [Scalar Galaxy.postman_collection.json](https://github.com/user-attachments/files/18021573/Scalar.Galaxy.postman_collection.json)

fixed with noAuth added in `authHelpers.ts`

**Comment by Mason-Little** [comment]
> i converted petstore openapi -> postman collection then import but i dont have any servers
> 
> image image
> [Swagger Petstore - OpenAPI 3.0.postman_collection.json](https://github.com/user-attachments/files/18021567/Swagger.Petstore.-.OpenAPI.3.0.postman_collection.json)

fixed with [updated Postman-to-openapi](https://github.com/scalar/scalar/pull/4091)

**Review by marclave — APPROVED** [review]

**Pull request merged**